src/Catalyst.Abstractions/Dfs/CoreApi/RepositoryData.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Catalyst.Abstractions.Dfs.CoreApi
+{
+    /// <summary>
+    ///   The statistics for <see cref="IStatsApi.RepositoryAsync"/>.
+    /// </summary>
+    public class RepositoryData
+    {
+        /// <summary>
+        ///   The number of blocks in the repository.
+        /// </summary>
+        /// <value>
+        ///   The number of blocks in the <see cref="IBlockRepositoryApi">repository</see>.
+        /// </value>
+        public ulong NumObjects;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)